src/IssueViz.UnitTests/Editor/SelectedIssueTagging/Highlighting/IssueHighlightViewTaggerProviderTests.cs

```diff
@@ -37,9 +35,8 @@ public class IssueHighlightViewTaggerProviderTests : CommonViewTaggerProviderTes
         public void MefCtor_CheckIsExported()
         {
             var aggregatorFactoryExport = MefTestHelpers.CreateExport<IBufferTagAggregatorFactoryService>(Mock.Of<IBufferTagAggregatorFactoryService>());
-            var locationServiceExport = MefTestHelpers.CreateExport<IAnalysisIssueSelectionService>(Mock.Of<IAnalysisIssueSelectionService>());
 
-            MefTestHelpers.CheckTypeCanBeImported<SelectedIssueLocationTaggerProvider, ITaggerProvider>(null, new[] { aggregatorFactoryExport, locationServiceExport });
+            MefTestHelpers.CheckTypeCanBeImported<IssueHighlightViewTaggerProvider, IViewTaggerProvider>(null, new[] { aggregatorFactoryExport });
```

Comment:
Miscellaneous: fixed the existing `IssueHighlighViewTaggerProvider` MEF test. I'd cut and pasted it from the corresponding `SelectedIssueLocationTaggerProvider` MEF test but not changed it, so it was testing the wrong type.

Reply:
I changed that my in PR as well 😆